Cargo.toml

```diff
@@ -5,7 +5,8 @@ authors = ["rhys <rhysormond@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-reqwest = { version = "0.10.0-alpha.2", features = ["json", "blocking"] }
+reqwest = { version = "0.10.0-alpha.2", features = ["json"] }
 serde = { version = "1.0.103", features = ["derive"] }
 serde_json = "1.0.44"
 chrono = "0.4.10"
+futures = "0.3.1"
```

Comment:
Literally just for `block_on`. I should see if the stdlib has that yet (or just write my own dirt-simple executor)